docs/data-sources/ondemandnodepool.md

```diff
@@ -0,0 +1,20 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "spot_ondemandnodepool Data Source - terraform-provider-spot"
+subcategory: ""
+description: |-
+  
+---
+
+# spot_ondemandnodepool (Data Source)
+
```

Comment:
Could you please add the description to the `templates/data-sources/ondemandnodepool.tmpl` file and then run `make generate` or `make docs`? This will ensure the documentation is populated in the file.